hosted/gke/helper/helper_cluster.go

```diff
@@ -445,6 +464,28 @@ func CreateGKEClusterOnGCloud(zone string, clusterName string, project string, k
 	return nil
 }
 
+// AddNodePoolOnGCloud adds a nodepool to the GKE cluster via gcloud CLI
+func AddNodePoolOnGCloud(clusterName, zone, project, npName string, extraArgs ...string) error {
+	if npName == "" {
+		npName = namegen.AppendRandomString("np")
```

Comment:
```suggestion
		npName = namegen.AppendRandomString("default-pool")
```
or change _default-pool_ to _np_ in https://github.com/rancher/hosted-providers-e2e/blob/main/cattle-config-provisioning.yaml for name consistency